### DISCUSSION TITLE
【灌水】这个新来的金发双马尾什么来头？

### DISCUSSION THREAD

**Discussion by yqj666** [share121/inter-knot#1715] [其他]
![mmexport1723447590547](https://github.com/user-attachments/assets/ae6a50b7-8f17-4591-a45f-6f92e0f88f67)
怎么好像露西的小猪很喜欢她啊
他们卡吕冬是人手一辆机车吗？

【【绝区零】肉腿！双马尾！卡吕冬新角色的柏妮思❤️-哔哩哔哩】 https://b23.tv/3NotVQd
![Uploading mmexport1723447584082.png…]()

**Comment by yqj666** [comment]
水

**Comment by GanlinSama** [comment]
说实在的，我看着幻视隔壁崩三小雀子

**Comment by ZhuYuan-148** [comment]
看标志像外环的人。
